hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyDeleteRequest.java

```diff
@@ -196,7 +201,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     case FAILURE:
       omMetrics.incNumKeyDeleteFails();
       LOG.error("Key delete failed. Volume:{}, Bucket:{}, Key{}. Exception:{}",
-          volumeName, bucketName, keyName, exception);
+          volumeName, bucketName, keyNameList, exception);
```

Comment:
same as above. we might only want to print the key that failed in the deletion instead of the whole list upon failures. print the whole list can be a debug or trace level log.